src/definitions/views/item.less

```diff
@@ -213,7 +213,7 @@
   line-height: @metaLineHeight;
   color: @metaColor;
 }
-.ui.items > .item .meta * {
+.ui.items > .item .meta *:not(time) {
```

Comment:
Adding a `>`, so `.ui.items > .item .meta > *` would be better. The `<time>` tag is just one example.